src/components/Calculator.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import './calculator.css';
+
+import Keys from './Keys';
+import calculate from './calculate';
+import Quote from './Quote';
+
+const Calculator = () => {
+  const [obj, setObj] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (btn) => {
+    setObj(calculate(obj, btn));
+  };
+
+  return (
+    <div className="calculator">
+      <div className="result">
+        <p>
+          {obj?.total}
+          {' '}
+          {obj?.operation}
+          {' '}
+          {obj?.next}
+        </p>
+      </div>
+      {[
+        ['AC', '+/-', '%', '÷'],
+        ['7', '8', '9', 'x'],
+        ['4', '5', '6', '-'],
+        ['1', '2', '3', '+'],
+        ['0', '.', '='],
+      ].map((row) => (
+        <ul key={`row${row[0]}`}>
+          {row.map((btn) => (
+            <Keys
+              key={btn}
+              cName={btn === '0' ? 'zero' : ''}
+              btn={btn}
+              clickHandler={() => handleClick(btn)}
+            />
+          ))}
+        </ul>
+      ))}
+      <Quote />
+    </div>
```

Comment:
- [ ] Your project is so good 😊 and works as expected, but when I took a look at the requirements I see 😃that The </ Quote> component should be imported into the main component and I see that you put the quote component in the calculator component, so please 💯 try to move the <Quote> component to the app component to stick to the requirements 🚀👌:
> Take a look at this snippet code 😊⤵️

```snippet

import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      {/* Rest of JSX */}
      <Calculator />
      <Quotes />
    </>
  )
}

export default App
```